### PR TITLE
update javascript examples attribute to 4.2.2

### PR DESCRIPTION
**Pull request by recrwplay**

### CODE REVIEW

File: jsManual/antora/antora.yml [hunk 1]
```diff
@@ -11,4 +11,4 @@ asciidoc:
     neo4j-buildnumber: '4.2'
     driver-version: "4.2"
     javascript-driver-version: "4.2.1"
```
Comment (martin-neotech) [inline jsManual/antora/antora.yml]: javascript-driver-version: "4.2.2"

https://github.com/neo4j/neo4j-javascript-driver/releases/tag/4.2.2

**Review by martin-neotech — CHANGES_REQUESTED** [review]
The latest javascript 4.2 driver is 4.2.2 https://github.com/neo4j/neo4j-javascript-driver/tags

File: jsManual/antora/antora.yml (hunk 1 above)
Comment (recrwplay) [reply jsManual/antora/antora.yml]: yes, good spot!
Done.

**Review by martin-neotech — APPROVED** [review]
Nice

**Pull request merged**